ale/drivers/lro_drivers.py

```diff
@@ -505,3 +505,59 @@ def spacecraft_direction(self):
         rotation = frame_chain.compute_rotation(1, lro_bus_id)
         rotated_velocity = spice.mxv(rotation._rots.as_dcm()[0], velocity)
         return rotated_velocity[0]
+
+
+class LroMiniRfIsisLabelNaifSpiceDriver(Radar, NaifSpice, IsisLabel, Driver):
+
+    @property
+    def wavelength(self):
+        """
+        Returns the wavelength in meters used for image acquistion. 
+
+        Returns
+        -------
+        : double
+          Wavelength in meters used to create an image
+        """
+
+        # Get float value of frequency in GHz
+        frequency = self.label['IsisCube']['Instrument']['Frequency'].value
+        wavelength = spice.clight() / frequency / 1000.0
+        return wavelength
+
+    @property
+    def scaled_pixel_width(self):
+        """
+        Returns the scaled pixel width
+
+        Returns
+        -------
+        : double
+          scaled pixel width
+        """
+        return self.label['IsisCube']['Instrument']['ScaledPixelHeight']; 
+
+    @property
+    def line_exposure_duration(self):
+        """
+        Line exposure duration in seconds. The sum of the burst and the delay for the return. 
+
+        Returns
+        -------
+        : double
+          scaled pixel width
+        """
+        return self.label['IsisCube']['Instrument']['LineExposureDuration']; 
+
+    @property
+    def range_conversion_coefficients(self):
+        """
+        Range conversion coefficients
+
+        Returns
+        -------
+        : List
+          range conversion coefficients
+        """
+        return self.label['IsisCube']['Instrument']['RangeCoefficientSet']; 
+
```

Comment:
What about the Reference Radii?

Reply:
The reference radii actually do come from the PCK so we don't need to grab them separately in here, the default works.